### PR TITLE
Added note about manual deletion of Teams cache to teams-for-vdi.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/8443.

@scanum @tiburd, please ask an SME to approve this, do not merge without someone else reviewing it. Thanks!

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by tiburd** [comment]
Sent email to a technical reviewer to review the changes in this PR.

File: Teams/teams-for-vdi.md [hunk 1]
```diff
@@ -127,6 +127,9 @@ Using Teams in a non-persistent setup also requires a profile-caching manager, f
 > [!NOTE]
 > A roaming folder (or, if you are using folder redirection, a caching manager) is required to ensure that the Teams app has the runtime data and files required to run the application. This is necessary to mitigate network latency issues or network glitches, which would otherwise cause application errors and a slow experience due to unavailable data and files.
 
+> [!IMPORTANT]
+> Manual deletion of the Teams cache is not supported due to a broken state in Teams. The only supported scenario is to sign out and uninstall the Teams client using the Windows Installer afterwards.
```
Comment (JohanFreelancer9) [inline Teams/teams-for-vdi.md:131]:
```suggestion
> Manual deletion of the Teams cache is not supported due to a broken state in Teams. The only supported scenario is to sign out and uninstall the Teams client using the Windows Installer afterward.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@dariomws - please commit the minor suggested change. Although the two words are generally interchangeable, Microsoft Style Guide prescribes the use of "afterward" rather than "afterwards". Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dstrome** [comment]
Checking with product team to validate change.

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request closed without merging**